### DISCUSSION TITLE
PDF Rendering ERROR - The system cannot find the path specified

### DISCUSSION THREAD

**Discussion by samrit2442** [quarto-dev/quarto-cli#4921] [Q&A] [answered]
Whenever I render the quarto book, website, or anything, I get this error (attached screenshot below); HTML output is created **CORRECTLY** but pdf output is not generated. But I want pdf output and HTML output both.
![image](https://user-images.githubusercontent.com/48163065/226955837-cc2a000a-a07c-4bfd-87fd-c88f7fa76281.png)

While rendering pdf only by clicking the "**Render PDF**" button from the menu bar, I get the error (attached below) in the console.

![image](https://user-images.githubusercontent.com/48163065/226957627-e6af9550-ad1a-44c4-99c3-bac7fbfa05ec.png)

I am using the latest versions of R, RStudio and Quarto. And, all I am doing this by creating an Rproject in a fresh directory.

**Comment by dragonstyle** [comment, marked as answer]
Apologies for this issue - the latest version of TexLive changed some internal paths and we have just released an updated version of Quarto 1.2  and 1.3 which include this fix.

Using either 1.2.475 or 1.3.283 should fix this issue. (https://quarto.org/docs/download/)

**Reply by samrit2442** [reply]
Thanks a lot!! It's working now.😊😊
